### ISSUE TITLE
Fix crashing when sound related endpoints are used in the control server

### ISSUE THREAD

**Issue by alok760** [fossasia/susi_installer#82]
#### Describe the bug
- When sound endpoints(eg- play,pause) are used in the control server, susi_linux crashes.
  
#### To Reproduce
- The endpoints are disabled for now.

#### Expected behavior
- Add back control playback functionality in the control server

**Comment by norbusan** [comment]
Have you found the **reason** for the crashes in susi linux? My guess this is an hardware/software interaction, because it happens randomly. What I have seen is that if the susi_linux client is started first, and **then** the soundserver, then I see less crashes - but then output will be done directly unless we add an online switch method.

**Comment by alok760** [comment]
@norbusan we can avoid the use of sound libraries in the soundserver altogether and insert the response to the state machine directly 
For example - clicking on `play` will send a play reply directly to the busy state.
what do you think?

**Comment by norbusan** [comment]
Yes, that was something I was thinking about, too, but didn't follow up till now since I don't know how to send messages to susi linux so that the state responds. How do you plan to inject these messages into susi linux?
